Source/WebKit/UIProcess/Automation/protocol/BidiBrowsingContext.json

```diff
@@ -176,6 +176,50 @@
                 { "name": "url", "type": "string" }
             ]
         },
+        {
+            "name": "navigationCommitted",
+            "description": "Event fired when navigation commits in a navigable.",
+            "spec": "https://w3c.github.io/webdriver-bidi/#event-browsingContext-navigationCommitted",
+            "parameters": [
+                { "name": "context", "$ref": "BidiBrowsingContext.BrowsingContext" },
+                { "name": "navigation", "$ref": "BidiBrowsingContext.Navigation", "nullable": true },
```

Comment:
It seems weird to me that a navigation can be nullable for a committed navigation. I looked into this and in the [introduction](https://w3c.github.io/webdriver-bidi/#module-browsingContext) to the `browsingContext` module it says:

> The [navigation id](https://html.spec.whatwg.org/multipage/browsing-the-web.html#navigation-id) for the navigation, or null when the navigation is canceled before making progress.

In order to make this more understandable on our end, I prefer we make `navigation` arguments non-`nullable` unless it can actually be null.

Reply:
Nit: consider renaming the `Navigation` type to `NavigationID` so it isn't ambiguous, or copy the `description` from other `navigation` parameters above.

I looked further up in this file, and we should fix up all `browsingContext.Navigation` parameters while we are here. For example, this return parameter appears multiple times:

```
{ "name": "navigation", "$ref": "BidiBrowsingContext.Navigation", "optional": true, "description": "The navigation ID, or null if not applicable." }
```

I think we should make this consistent- can you update the `optional` to `nullable` in those cases?